common/scripted_triggers/wc_his_character_triggers.txt

```diff
@@ -129,12 +141,48 @@ is_kilrogg_trigger = {
 	exists = character:10005 # Kilrogg
 	this = character:10005 # Kilrogg
 }
+is_drekthar_trigger = {
+	exists = character:10100 # Drek'thar
+	this = character:10100 # Drek'thar
+}
+is_draka_trigger = {
+	exists = character:10020 # Draka
+	this = character:10020 # Draka
+}
+is_nazgrel_trigger = {
+	exists = character:10658 # Nazgrel
+	this = character:10658 # Nazgrel
+}
 
 # Dwarves
 is_magni_trigger = {
 	exists = character:17001 # Magni Bronzebeard
 	this = character:17001 # Magni Bronzebeard
 }
+is_muradin_trigger = {
+	exists = character:17002 # Muradin Bronzebeard
+	this = character:17002 # Muradin Bronzebeard
+}
+is_brann_trigger = {
+	exists = character:17028 # Brann Bronzebeard
+	this = character:17028 # Brann Bronzebeard
+}
+is_moira_trigger = {
```

Comment:
What did you use as Moira's reference? I feel like she should have a wider head.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120846044-a6cff900-c582-11eb-8c2a-dcfd3caec486.png)
![image](https://user-images.githubusercontent.com/46384992/120846086-b4857e80-c582-11eb-9662-08d61d42ace7.png)

</details>

Reply:
Mostly [this](https://static.wikia.nocookie.net/hearthstone_gamepedia/images/8/8a/Moira_Bronzebeard_full.jpg/revision/latest/scale-to-width-down/420?cb=20150816011430). I can widen her head a bit if you want.

Reply:
> Mostly [this](https://static.wikia.nocookie.net/hearthstone_gamepedia/images/8/8a/Moira_Bronzebeard_full.jpg). I can widen her head a bit if you want.

She lacks dwarven features. I would make her head even wider. Top head lower. Nose bigger... it should resemble a potato... like it's on your art.